k8s-config/deploy-mobi.yml

```diff
@@ -25,6 +25,9 @@ spec:
       containers:
       - image: acend/kubernetes-basics-training:mobi
         imagePullPolicy: Always
+        env:
```

Comment:
I'm not sure whether that works as intended. I think the ENV is used during docker build time and not as configuration in the deployment.